lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9192,6 +9192,14 @@ static bool SwiftifiableCAT(const clang::CountAttributedType *CAT) {
   return CAT && CountedByExpressionValidator().Visit(CAT->getCountExpr());
 }
 
+static bool SwiftifiablePointerType(Type swiftType) {
+  // don't try to transform any Swift types that _SwiftifyImport doesn't know how to handle
+  Type nonnullType = swiftType->lookThroughSingleOptionalType();
+  PointerTypeKind PTK;
+  return nonnullType->isOpaquePointer() ||
+    (nonnullType->getAnyPointerElementType(PTK) && PTK != PTK_AutoreleasingUnsafeMutablePointer);
```

Comment:
Since you are already checking for and allowing types imported as opaque pointers here, can you add some test cases to make sure those work?

Reply:
We have a test case of that here: https://github.com/swiftlang/swift/blob/f57e2d7676e18f513afb219dbc1c2c738af4e019/test/Interop/C/swiftify-import/Inputs/sized-by.h#L23